### ISSUE TITLE
Inserting Instant failed

### ISSUE THREAD

**Issue by max5599** [tpolecat/doobie#1096]
Since the version 0.8.7 my queries are failing when trying to insert an instant like `sql"insert into test_instant(my_instant) values (${Instant.now()})".update.run`.
It seems that that value is not retrieved:
```
Data truncation: Incorrect datetime value: '\xAC\xED\x00\x05sr\x00\x0Djava.time.Ser\x95]\x84\xBA\x1B"H\xB2\x0C\x00\x00xpw\x0D\x02\x00\x00\x00\x00]\xF7\x9A\xC4;m\x03@x' for column 'my_instant' at row 1
```

**Comment by tpolecat** [comment]
This is related to #1095 … we may need to roll back that change or provide a different instance. For now probably stick with 0.8.6

**Comment by tpolecat** [comment]
Fixed in 0.8.8, see https://github.com/tpolecat/doobie/releases
